src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -289,6 +293,12 @@ export const ChannelInput: React.FC<IChannelInput> = ({
 
   const mentionSelectAction = (e: React.MouseEvent | React.KeyboardEvent) => {
     e.preventDefault()
+    if (refSelected?.current && refMentionsToSelect?.current) {
+      if (refMentionsToSelect.current[refSelected.current]) {
+
```

Comment:
something is werid here, these `if`'s are added, but they don't do anything

Reply:
ou i forgot about it